integration/spark/runtest.sh

```diff
@@ -28,6 +28,9 @@ popd
 export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
 export MAVEN_OPTS="${MAVEN_OPTS} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
 
+echo "Patching Spark"
+patch -d /spark -p1 -i /arrow/integration/spark/ARROW-6429.patch
```

Comment:
@kszucs I thought this would be better in the Dockerfile after checking out spark, but for some reason the patch didn't seem to apply. I was trying this and it seemed to run, but the file wasn't patched
```
COPY integration/spark/spark-arrow-java-IpcOption.patch /spark/
RUN patch -d /spark -p1 -i /tmp/spark-arrow-java-IpcOption.patch
```
Any ideas?

Reply:
Nevermind, it seems to work now. I'll change it..